src/libraries/System.Security.Cryptography.Algorithms/tests/AesCcmTests.cs

```diff
@@ -678,6 +678,17 @@ public static IEnumerable<object[]> GetNistCcmTestCasesWithNonEmptyPT()
 
     public class AesCcmIsSupportedTests
     {
+        public static bool RuntimeSaysIsNotSupported => !AesCcm.IsSupported;
+
+        [ConditionalFact(nameof(RuntimeSaysIsNotSupported))]
```

Comment:
I don't know if these tests will get exercised during CI or not, but I confirmed they work locally without openssl.